en/micro-integrator/mkdocs.yml

```diff
@@ -522,6 +522,19 @@ nav:
           - 'SMPP Reference': 'references/connectors/smpp-connector/smpp-connector-config.md'
           - 'SMPP Inbound Endpoint Example': 'references/connectors/smpp-connector/smpp-inbound-endpoint-example.md'
           - 'SMPP Inbound Endpoint Reference': 'references/connectors/smpp-connector/smpp-inbound-endpoint-config.md'
+        - Amazon DynamoDB Connector:
```

Comment:
I think this is already added. Please check.

Reply:
Yes already added.Updated.